CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Master
+
+* Adds initializing android sdk using `Instabug.start` instead of inside the application class
```

Comment:
```suggestion
* You can now initialize Instabug on Android by calling `Instabug.start` from Javascript, instead of calling the builder method from the application class.
```